### PR TITLE
chore(setup): add support for mulitple languages

### PR DESCRIPTION
**Pull request by MikeManzi**
#### What does this PR do?

- Adds support for multiple languages(English & French)

#### Description of Task to be completed?

- User can be able to change the language

#### How should this be manually tested?

- Clone the repo

- checkout to branch `ch-support-multiple-languages`

- Run `npm install` to install modules

- Run `npm run dev` to start the development server

- In your API testing tool (postman recommended) pass language(en/fr) in the Accept-Language header and call API `http://localhost:4000/`

#### Background context

- The default language is English

#### What are the relevant Trello stories?

- [Link to my implemented Trello story](https://trello.com/c/OdyIXu6M/20-support-multiple-languages)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -30,12 +32,11 @@ app.use(
  *         description: Server status and welcome message
  */
 app.get("", (req: Request, res: Response) => {
-  return res.status(200).send({
-    message: "Welcome to Phantom server",
-    serverStatus: "RUNNING",
-  });
+    // req.setLocale("fr");
```
Comment (dmutoni) [inline src/index.ts]: Why these comments?

Comment (MikeManzi) [reply src/index.ts]: I was testing with that however, it is now removed

**Review by degide — CHANGES_REQUESTED** [review]
@MikeManzi Remove the unnecessary back slash `\` at the end of line 8 in `src/index.ts`

**Review by degide — APPROVED** [review]

**Review by kelliaUmuhire — APPROVED** [review]

**Review by degide — CHANGES_REQUESTED** [review]
It doesn't work. Tried changing between `en and `fr` in the `Accept-Language` header on the welcome endpoint and no change.

**Review by degide — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (dmutoni) [reply src/index.ts]: Yeah thank you

**Review by dmutoni — APPROVED** [review]

**Pull request merged**